pkg/updater/resultstore/resultstore.go

```diff
@@ -178,6 +179,72 @@ func ResultStoreColumnReader(client *DownloadClient, reprocess time.Duration) up
 	}
 }
 
+func parseTaxonomizer(msg string) (string, string) {
+	parts := strings.SplitN(msg, "~", 2) // class_type~message
+	if len(parts) != 2 {
+		return "", ""
+	}
+	return parts[0], parts[1]
+}
+
+// cellMessageIcon attempts to find an interesting message and icon by looking at the associated properties and tags.
+func cellMessageIcon(annotations []*configpb.TestGroup_TestAnnotation, properties map[string][]string, tags []string) (string, string) {
+	check := make(map[string]string, len(properties))
+	for k, v := range properties {
+		check[k] = v[0]
+	}
+
+	tagMap := make(map[string]string, len(annotations))
+
+	checkOrder := make([]string, 0, len(annotations))
+
+	for _, a := range annotations {
+		k := a.GetPropertyName()
+		check[k] = a.ShortText
+		tagMap[k] = a.ShortText
+		checkOrder = append(checkOrder, k)
+	}
+
+	for _, key := range checkOrder {
```

Comment:
checkOrder is now just iterating over the annotations in order, so I think the logic above can be simplified.

Reply:
Think this is still true; checkOrder == annotations, so instead of creating and appending to a new slice we can remove checkOrder altogether and just use annotations in this loop.

Reply:
Resolved